microbench/db_basic_bench.cc

```diff
@@ -320,7 +320,7 @@ static void DBPutArguments(benchmark::internal::Benchmark* b) {
       {"comp_style", "max_data", "per_key_size", "enable_statistics", "wal"});
 }
 
-static const uint64_t DBPutNum = 10l << 10;
+static const uint64_t DBPutNum = 20l << 10;
```

Comment:
Nit: `static constexpr uint64_t DBPutNum = 20ull << 10`. `l` can be 32bit.

Reply:
The preferred way is uint64_t{20}